files/timrjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "timrjs",
+  "repo": "joesmith100/timrjs",
+  "files": {
+    "basePath": "dist/",
+    "include": ["timr.js", "timr.min.js"]
```

Comment:
No, but the dev version contains more descriptive errors. So would be nice to be able to offer the option to people.